conftest.py

```diff
@@ -71,6 +71,13 @@ def pytest_addoption(parser):
         default=20,
         help="Max lines to output in a ssh log (0 if no limit)"
     )
+    parser.addoption(
+        "--sr-disk",
+        action="append",
+        default=[],
+        help="Name of an available disk (sdb) or partition device (sdb2) to be formatted and used in storage tests. "
+             "Set it to 'auto' to let the fixtures auto-detect available disks."
```

Comment:
`auto` as default value?

Reply:
This would make the parameter not mandatory anymore.